python/hail/expr/functions.py

```diff
@@ -702,12 +702,19 @@ def locus(contig, pos, reference_genome: Union[str, ReferenceGenome] = 'default'
 
 @typecheck(global_pos=expr_int64,
            reference_genome=reference_genome_type)
-def locus_from_global_position(global_pos, reference_genome: Union[str, ReferenceGenome] = 'default') -> LocusExpression:
-    """Construct a locus expression from a global position and a reference genome.
+def locus_from_global_position(global_pos,
+                               reference_genome: Union[str, ReferenceGenome] = 'default') -> LocusExpression:
+    """Constructs a locus expression from a global position and a reference genome.
     The inverse of :meth:`.LocusExpression.global_position`.
 
     Examples
     --------
+    >>> hl.locus_from_global_position(0).value
```

Comment:
New line after `-------`.

Reply:
oops, merged without seeing comments. the newline after Examples/------ isn't necessary for rendering. Do you think it looks better? I've come to think that the numpy style is vertically spaced enough as is, so we don't need extra newlines everywhere. Others could have different opinions though!